src/main/webapp/form.jsp

```diff
@@ -38,63 +38,40 @@
                     <div class="values-row">
                         <div class="value-header">X</div>
                         <div class="value-choose-block">
-                            <select name="xVal">
-                                <option disabled>Выберите значение X</option>
-                                <option value="-5">-5</option>
-                                <option value="-4">-4</option>
-                                <option value="-3">-3</option>
-                                <option value="-2">-2</option>
-                                <option value="-1">-1</option>
-                                <option value="0">0</option>
-                                <option value="1">1</option>
-                                <option value="2">2</option>
-                                <option value="3">3</option>
-                                <option value="4">4</option>
-                                <option value="5">5</option>
-                            </select>
+                            <input type="hidden" id="selectedValue" name="xVal" value="" />
```

Comment:
Bettter find how for - tag works and use it if it eally does it. It gives shorter and consice code .... 
<c:for -4 ...4 > etc...